### PR TITLE
SG-34675 - Rebranding

### PR DESCRIPTION
**Pull request by juanburgosautoglb**
Change all the shotgrid/shotgun occurrences in the repository.

Update logos.

### CODE REVIEW

File: info.yml [hunk 1]
```diff
@@ -14,8 +14,8 @@ requires_shotgun_fields:
 
 
 # More verbose description of this item
-display_name: "ShotGrid Engine for ShotGrid Desktop v2"
-description: "ShotGrid Support in ShotGrid Desktop v2"
+display_name: "Flow Production Tracking Engine for Desktop app v2"
```
Comment (carlos-villavicencio-adsk) [inline info.yml:17]: Should this be "... Engine for Create app" now? @geffrak @bernie-laberge can you confirm this?

File: python/tk_desktop2/action_handler.py [hunk 2]
```diff
@@ -581,7 +584,8 @@ def _execute_action(self, path, action_str):
             if self.KEY_PICKLE_STR not in json_obj:
                 raise RuntimeError(
                     "The command's serialized Python data could not be found in the action's payload"
-                    "that ShotGrid Create provided. The action cannot be executed as a result."
+                    "that Flow Production Tracking Create provided. The action cannot be executed as "
```
Comment (carlos-villavicencio-adsk) [inline python/tk_desktop2/action_handler.py:587]: I'm not sure if "Flow Production Tracking Create" is correct 🤔

**Review by carlos-villavicencio-adsk — COMMENTED** [review]
Hello @geffrak, @bernie-laberge, is any of you able to help us with a code review of this rebranding PR? Thank you.

File: info.yml (hunk 1 above)
Comment (geffrak) [reply info.yml:17]: `Engine for Flow Production Tracking Create` ?

File: python/tk_desktop2/action_handler.py (hunk 2 above)
Comment (geffrak) [reply python/tk_desktop2/action_handler.py:587]: It's the full name

**Review by geffrak — APPROVED** [review]

**Pull request merged**